### ISSUE TITLE
what‘s the meaning of INFERENCE_IMAGE_PATH

### ISSUE THREAD

**Issue by jiachen0212** [yangxue0827/R2CNN_FPN_Tensorflow#19]
thanks very much for your code。 When i run the train1.py, i always meet this bug：
UnknownError (see above for traceback): error: /home/travis/miniconda/conda-bld/conda_1485299288502/work/opencv-3.2.0/modules/imgproc/src/rotcalipers.cpp:166: error: (-215) orientation != 0 in function rotatingCalipers
So I wonder if it's because I didn't configure the  INFERENCE_IMAGE_PATH folder?  
hope you can give a light!

**Comment by yangxue0827** [comment]
You only need this parameter when you run the demo,py. It represents the path of demo images.

发送自 Windows 10 版邮件应用

发件人: jiachen0212
发送时间: 2018年4月1日 21:17
收件人: yangxue0827/R2CNN_FPN_Tensorflow
抄送: Subscribed
主题: [yangxue0827/R2CNN_FPN_Tensorflow] what‘s the meaning of INFERENCE_IMAGE_PATH (#19)

thanks very much for your code。 When i run the train1.py, i always meet this bug：
UnknownError (see above for traceback): error: /home/travis/miniconda/conda-bld/conda_1485299288502/work/opencv-3.2.0/modules/imgproc/src/rotcalipers.cpp:166: error: (-215) orientation != 0 in function rotatingCalipers
So I wonder if it's because I didn't configure the INFERENCE_IMAGE_PATH folder?
hope you can give a light!

**Comment by jiachen0212** [comment]
yeah   but what about the bug, i can not run the train1.py successfully。。。
UnknownError (see above for traceback): error: /home/travis/miniconda/conda-bld/conda_1485299288502/work/opencv-3.2.0/modules/imgproc/src/rotcalipers.cpp:166: error: (-215) orientation != 0 in function rotatingCalipers

**Comment by jiachen0212** [comment]
i update the code and then the bug solved/.....maybe something wrong in the convert_data_to_tfrecord.py .......
anyway, thanks very much for your codes!
